src/lib/ToastMessage.svelte

```diff
@@ -0,0 +1,14 @@
+<p>Map copied</p>
```

Comment:
Since this is not a reusable component, should it just be in the modal?
That way you can avoid the store too

Reply:
done

Reply:
fixed